### ISSUE TITLE
error with `tyscan init`

### ISSUE THREAD

**Issue by gianpaj** [sider/TyScan#125]
```
npx tyscan init
Error: ENOENT: no such file or directory, copyfile '.../node_modules/tyscan/dist/cli/subcommand/../../../sample/tyscan.yml
    at Object.copyFileSync (fs.js:1790:3)
    at InitCommand.run (/Users/gpalumbo/github/neo-pwa/client/node_modules/tyscan/dist/cli/subcommand/initCommand.js:8:12)
    at Object.runCommand (/Users/gpalumbo/github/neo-pwa/client/node_modules/tyscan/dist/cli/runner.js:6:30)
    at Command.<anonymous> (/Users/gpalumbo/github/neo-pwa/client/node_modules/tyscan/dist/cli/main.js:32:30)
    at Command.listener (/Users/gpalumbo/github/neo-pwa/client/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:223:5)
    at Command.parseArgs (/Users/gpalumbo/github/neo-pwa/client/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/gpalumbo/github/neo-pwa/client/node_modules/commander/index.js:474:21)
    at Object.main (/Users/gpalumbo/github/neo-pwa/client/node_modules/tyscan/dist/cli/main.js:15:19)
    at Object.<anonymous> (/Users/gpalumbo/github/neo-pwa/client/node_modules/tyscan/dist/cli/bin.js:4:8)
```

**Comment by ybiquitous** [comment]
@gianpaj Thanks for the bug report! I could reproduce it.

I will fix it soon.

**Comment by ybiquitous** [comment]
I've published v0.3.2 including this bugfix! 🎉 

https://www.npmjs.com/package/tyscan/v/0.3.2
https://github.com/sider/TyScan/releases/tag/v0.3.2

**Comment by gianpaj** [comment]
great thanks!
